careers/working-at-cfpb/index.html

```diff
@@ -0,0 +1,150 @@
+            {# TODO: Replace placeholder div with image. #}
+            <div class="hero_img"
+                 style="width:1200px;height:300px;background-image:url({{ 'http://placehold.it/1200x300.jpg ' }});">
+            </div>
+        </div>
+    </section>
+{%- endblock %}
+
+{% block content_main %}
+
+    <section class="block
+                    block__flush-top
+                    block__sub">
+            <h1>Working at the CFPB</h1>
+            <p class="h3">
+                Our mission stems from our commitment to serving the consumer
+                and our country.
+                Our vision is a consumer finance marketplace that works for
+                American consumers, responsible providers,
+                and the economy as a whole.
+            </p>
+            <p class="short-desc">
+                This vision can only come from the people who are working
+                toward it.
+                Creating this kind of market takes a dedication to service,
+                leadership and innovation.
+            </p>
+    </section>
+
+    {% import "macros/mobile-carousel.html" as mobile_carousel %}
+    {{ mobile_carousel.render({
+        'image_sizes_class': 'media_image__100',
+        'image_base_url':    '/static/img/',
+        'data': [{
+            'image_filename': 'serve.png',
+            'title':          'Serve',
+            'desc':      'Our mission begins with service to the
+                          consumer and our country.
+                          We serve our colleagues by listening to
+                          one another and by sharing our collective
+                          knowledge and experience.'
+            }, {
+            'image_filename': 'lead.png',
+            'title':          'Lead',
+            'desc':      'Fostering leadership and collaboration at
+                          all levels is at the core of our success.
+                          We believe in investing in the growth of
+                          our colleagues and in creating an
+                          organization that is accountable to the
+                          American people.'
+            }, {
+            'image_filename': 'innovate.png',
+            'title':          'Innovate',
+            'desc':      'Our organization embraces new ideas
+                          and technology.
+                          We are focused on continuously improving,
+                          learning, and pushing ourselves to be great.'
+            }]
+        })
+    }}
+
+    <div class="block
+                block__flush-top">
+        <p class="short-desc">As a bureau, we support these values by focusing on equal opportunity, diversity, and inclusion. We value candidates from all backgrounds, traditional and non-traditional. We believe in investing in the growth of our colleagues and in creating an organization that is accountable to and representative of the American people.</p>
```

Comment:
> These are no longer descriptions. Maybe we should pull this style for wider re-use or rename the class something more descriptive?

Yeah I'm not sure what to do here. This appeared to be the style needed to match the mocks. Does anyone know of a different one that fits here? Needs to stylistically look like:

![screen shot 2015-06-30 at 10 49 52 am](https://cloud.githubusercontent.com/assets/704760/8433768/ca66563a-1f15-11e5-93f5-77c154dc41b1.png)

cc @Scotchester @sebworks @KimberlyMunoz 

> Add semantic line breaks here and on L:110

How about this? https://github.com/cfpb/cfgov-refresh/commit/8bca73f22280940c715d46f5787eb04a9c6b1ec3